### PR TITLE
Add loaddata command and some related changes

### PR DESCRIPTION
**Pull request by allait**
Downloads a database dump from S3 and runs it on the environment RDS instance.

Runs Elasticsearch import script from the digitalmarketplace-api repository with environment API and Elasticsearch as arguments.

A lot of this will change once we have write endpoints on api and search-api that would allow loading data throught them instead of using the databases directly.

### CODE REVIEW

**Comment by robyoung** [comment]
Some of the commit messages could benefit from a bit of explanation to why they're being done. For example, using the `KeyName` type; was it causing a specific problem or was this just to use a more specific type? [git styleguide](https://github.com/alphagov/styleguides/blob/master/git.md)

**Comment by allait** [comment]
@robyoung no idea how it's actually different from String tbh, I saw there was a separate type for keys, so I changed the templates. I've read the style guide before.

**Comment by robyoung** [comment]
That in itself is useful information.

File: dmaws/commands/loaddata.py [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+        es_endpoint = 'http://{}:9200/services/'.format(elasticsearch_ip)
+        requests.put(es_endpoint)
+
+        ctx.log('Inserting data into Elasticsearch')
+        run_cmd([
+            './scripts/process-g6-into-elastic-search.py',
+            es_endpoint,
+            plan.get_value('stacks.api.outputs.URL') + '/services',
+            plan.get_value('api.auth_tokens')[0]
+        ], cwd=api_repo_path, ignore_errors=True)
```
Comment (robyoung) [inline dmaws/commands/loaddata.py]: Why do we ignore errors here?

Comment (allait) [inline dmaws/commands/loaddata.py]: There's no step that depends on this being successful, and we still need to remove downloaded file and do the dev-access teardown

File: dmaws/commands/loaddata.py [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+        ], ignore_errors=True)
+
+    if 'search' in tasks:
+        ctx.log('Creating Elasticsearch index')
+        es_endpoint = 'http://{}:9200/services/'.format(elasticsearch_ip)
+        requests.put(es_endpoint)
+
+        ctx.log('Inserting data into Elasticsearch')
+        run_cmd([
+            './scripts/process-g6-into-elastic-search.py',
```
Comment (robyoung) [inline dmaws/commands/loaddata.py]: Where is this script from?

Comment (allait) [inline dmaws/commands/loaddata.py]: API repo path, which we take as an argument. It'll get replaced with something more suitable as soon as the API is pushing updates to Elasticsearch

File: dmaws/commands/loaddata.py (hunk 1 above)
Comment (robyoung) [inline dmaws/commands/loaddata.py]: I think it would still be useful to know that it has failed.

Comment (allait) [inline dmaws/commands/loaddata.py]: The command output is printed in any case

File: dmaws/commands/loaddata.py (hunk 2 above)
Comment (robyoung) [inline dmaws/commands/loaddata.py]: Ah, I see, missed the `cwd`.

File: dmaws/commands/loaddata.py (hunk 1 above)
Comment (robyoung) [inline dmaws/commands/loaddata.py]: It swallows the fact that there was any error. If the command being run doesn't output anything you have no feedback that it failed. A log message to say that the command has failed would be helpful.

Comment (allait) [inline dmaws/commands/loaddata.py]: Added start and finish log messages to run_cmd. Prints the command name only and return code on exit.

**Comment by allait** [comment]
Rebased and fixed merge conflict on KeyName type change

**Pull request merged**